test/linalg/svd.jl

```diff
@@ -4,6 +4,31 @@ using Base.Test
 
 using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
+@testset "Simple svdvals / svdfact tests" begin
+    ≈(x,y) = isapprox(x,y,rtol=1e-15)
```

Comment:
> Dangit, you're right, sorry

No problem at all.  I guess this would be a natural approach (perhaps?) in a language with efficient dynamic scoping, eg flisp's `with-bindings` :-)